docs/integrations/destinations/postgres.md

```diff
@@ -127,6 +127,10 @@ characters.
     :::
 12. Click **Set up destination**.
 
+## ⚠️ Warning
```

Comment:
maybe use the `:::warning` thing?

Reply:
Today I learned that this is called an "admonition" https://docusaurus.io/docs/next/markdown-features/admonitions

I went with ":::caution"

<img width="1722" alt="Screenshot 2023-05-05 at 9 02 15 AM" src="https://user-images.githubusercontent.com/303226/236508982-68ab5555-4a3e-466b-af8f-fe567c5050d5.png">